docs/data-guidelines.md

```diff
@@ -166,16 +166,29 @@ For example, although the UI Events specification defines a [`DOM_KEY_LOCATION_S
 
 This guideline was proposed in [#7936](https://github.com/mdn/browser-compat-data/issues/7585), based in part on previous discussion in [#7585](https://github.com/mdn/browser-compat-data/issues/7585).
 
-## Release lines and backported features
+## Release lines
```

Comment:
I don't think we should change the existing guideline. It applies to all browsers, not just Safari. I think this has come up for other browsers (perhaps when `SharedArrayBuffer` was removed?), though it's definitely less prominent.